src/main/java/com/impactupgrade/nucleus/service/segment/StripeDataSyncService.java

```diff
@@ -29,20 +39,77 @@ public void init(Environment env) {
 
   @Override
   public void syncContacts(Calendar updatedAfter) throws Exception {
+    if (!env.getConfig().stripe.enableContactSync) {
+      return;
+    }
+
+    // TODO: also sync getDonorOrganizationAccounts
```

Comment:
@VSydor Maybe we should go ahead and implement this? Only concern is business names and billing addresses can be a little more finicky than individuals...